javascript/javascript1/week1/agecalculator.js

```diff
@@ -0,0 +1,7 @@
+// Both  yearOfBirth and yearFuture variables will be constant because they are not changeable
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+
+let age = yearFuture - yearOfBirth;
```

Comment:
you could also use const to declare `age`, because you will not reassign it later.